config/config_test_reload_error_test.go

```diff
@@ -17,31 +18,31 @@ func TestErrorReloading(t *testing.T) {
 	assert.NoError(t, err)
 	defer os.RemoveAll(tmpDir)
 
-	rulesFile, err := ioutil.TempFile(tmpDir, "*.toml")
```

Comment:
It's kinda lame, but I feel like we should load and run tests against both if we continue to support both file formats. Wouldn't want to let the toml config bitrot when the majority of our customers (for quite a while) will be using it.